tests/test_envs.py

```diff
@@ -0,0 +1,33 @@
+import gym
+import pytest
+
+from imitation.envs import ENV_NAMES
+from imitation.util import make_vec_env
+
+PARALLEL = [False, True]
+
+try:
+  import mujoco_py as _
+  del _
+  MUJOCO_OK = True
+except ImportError:
+  MUJOCO_OK = False
+
+
+@pytest.mark.skipif(not MUJOCO_OK,
+                    reason="Requires `mujoco_py`, which isn't installed.")
+@pytest.mark.parametrize("parallel", PARALLEL)
+def test_envs(parallel):
+  """Check that our custom environments don't crash on `step`, and `reset`."""
+  for env_name in ENV_NAMES:
+    if parallel:
```

Comment:
`parallel` is misleading here as it suggests true parallelism, i.e. `SubprocVecEnv`, but `make_vec_env` can also make `dummy_vec_env`. I'd suggest renaming as `vector`.

I'm also not sure why we need to test both. The *environment* doesn't know if it's in a `VecEnv` or not. So can probably just test one or the other.

Reply:
That sounds right, I'll remove parallel if I keep this test